File: 0414-third-maximum-number/0414-third-maximum-number.cs
public class Solution {
    public int ThirdMax(int[] nums) {
   List <int> distinctSorted=nums.Distinct().OrderByDescending(x=>x).ToList();
        if(distinctSorted.Count<3){
            return distinctSorted[0];
        }else{
            return distinctSorted[2];
        }
    }
}

File: 0414-third-maximum-number/0414-third-maximum-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var thirdMax = function(nums) {
    let a=nums.sort((a,b)=>b-a)
    let b=Array.from(new Set(a))
        if(b.length<3){
            return b[0]
        }
        else{
            return b[2]
        }
    
};